reader.go

```diff
@@ -178,10 +179,10 @@ func (lr *LineReader) processInput(input string, n *NilShell) ProcessingCode {
 			copy(buffer, lr.lastSearchText)
 			lr.buffer = buffer
 		}
-		fmt.Printf("\r\n")
+		fmt.Printf("\n")
 		return CodeComplete
 	case KEY_CTRL_C:
-		fmt.Printf("\r\n")
```

Comment:
Why do we not want to do a carriage return here too ?

Reply:
b/c we're automatically resetting the cursor position when we reset the prompt.  seemed kind of redundant.

Reply:
also, it would require us to eliminate flicker here as well, since it's moving the cursor to a low X axis location

Reply:
Ah gotcha.